Refit/RequestBuilderImplementation.cs

```diff
@@ -666,6 +675,11 @@ private async Task<T> DeserializeContentAsync<T>(HttpResponseMessage resp, HttpC
                     }
                 }
 
+                foreach (var property in propertiesToAdd)
+                {
+                    ret.Properties.Add(property.Key, property.Value);
```

Comment:
I know there's a test for the Dup, but Add won't throw here if the key is already present?

Reply:
I've updated this to use the indexer rather than Add method. In the first instance though propertiesToAdd never contains duplicate keys. Also since Refit itself is constructing the HttpRequestMessage before any DelegatingHandlers get to run it's going to be empty, so I think it's likely OK, but have changed it to more of a 'last write wins' in keeping with how the rest of the feature works and erring on the side of not blowing up if it ever happens.